app/views/organisations/index.html.erb

```diff
@@ -1,33 +1,35 @@
-<h1 class="govuk-heading-l">Signed Up Organisations: <%= @all_organisations.count %></h1>
+<h1 class="govuk-heading-l">All organisations</h1>
+<h3 class="govuk-heading-m">
+  <%= pluralize(@all_organisations.count, 'organisation') %> are signed up to GovWifi
```

Comment:
Might not even be necessary to pluralize here, given I've assumed "are" rather than "is". There'll always be more than one!